scripts/deep_learning/clusterSplitting/Datasets.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+import torch
+from torch.utils.data import Dataset
+import sys
+
+################################################################################################################################################
+################################################################################################################################################
+
+class ClassificationDataset(Dataset):
```

Comment:
Could do with some comments to describe what this class represents

Reply:
done